### PR TITLE
Issue 418 Increase default SQS retention to maximum of 4 days

### PR DESCRIPTION
**Pull request by payman81**
Link to issue #418

### CODE REVIEW

**Comment by martincostello** [comment]
Fixes #418.

**Comment by slang25** [comment]
Deja vu: https://github.com/justeat/JustSaying/pull/272

**Comment by slang25** [comment]
I think we should default to the SQS default which is 4 days

**Comment by slang25** [comment]
Inline with version 4: https://github.com/justeat/JustSaying/blob/v4/JustSaying.AwsTools/JustSayingConstants.cs#L42

This must have got lost when the v4 commits were picked across into v5, that's a worry.

File: JustSaying/AwsTools/JustSayingConstants.cs [hunk 1]
```diff
@@ -41,7 +41,7 @@ public static class JustSayingConstants
         /// <summary>
         /// Default message retention period on a queue in seconds
         /// </summary>
-        public static int DefaultRetentionPeriod => 60 * 10;    //10 minutes
+        public static int DefaultRetentionPeriod => MaximumRetentionPeriod;    //14 days
```
Comment (slang25) [inline JustSaying/AwsTools/JustSayingConstants.cs]:
```suggestion
        public static int DEFAULT_RETENTION_PERIOD = 345600;    //4 days
```

**Comment by payman81** [comment]
Greetings old friends 👋

**Comment by martincostello** [comment]
Looks like this broke a test.

**Comment by slang25** [comment]
Sorry, I made an invalid suggestion, and updated it 3 times in quick succession to correct it. It must have been accepted while it was invalid. A race condition in a review 😆

**Pull request merged**